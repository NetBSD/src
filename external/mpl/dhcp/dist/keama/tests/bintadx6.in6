# binary-to-ascii data expression

# empty configs are not accepted by Kea
default-lease-time 1800;

# reduce literals
class "literal" {
    match if option dhcp6.client-data = binary-to-ascii(16, 8, "-", "\007foo");
}
