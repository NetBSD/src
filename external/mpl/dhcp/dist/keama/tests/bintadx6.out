{
  # binary-to-ascii data expression
  # empty configs are not accepted by Kea
  "Dhcp6": {
    "valid-lifetime": 1800,
    "client-classes": [
      # reduce literals
      {
        "name": "literal",
        /// from: match if (option dhcp6.client-data) = (binary-to-ascii(16, 8, '-', 'foo'))
        "test": "option[45].hex == '7-66-6f-6f'"
      }
    ]
  }
}
